python/ray/autoscaler/_private/command_runner.py

```diff
@@ -32,6 +32,7 @@
 HASH_MAX_LENGTH = 10
 KUBECTL_RSYNC = os.path.join(
     os.path.dirname(os.path.abspath(__file__)), "kubernetes/kubectl-rsync.sh")
+RAY_HOME = "/home/ray"  # ray home path in the container image
```

Comment:
hmm, can you move this to `_private/constants.py`?

Reply:
Moved!